sw/SoC/Makefile

```diff
@@ -0,0 +1,39 @@
+# Author: Stefano Mercogliano 	<stefano.mercogliano@unina.it>
+# Description:
+#   This is a global makefile for the System-on-Chip software projects.
+#	It can be used to compile all examples and create new projects based 
+#	on our provided template.
+
+# Environment check
+ifndef ROOT_DIR
+$(error Setup script settings.sh has not been sourced, aborting)
+endif
+
+EXAMPLES_NAMES	:= $(shell basename --multiple $(SW_ROOT)/SoC/examples/*)
+EXAMPLES_LIST	:= $(foreach ip,${EXAMPLES_NAMES},$(SW_ROOT)/SoC/examples/${ip})
+
+PROJECT_NAME ?= dummy
+
+all: cbuild_examples
+
+# Create a new project out of the template
+create_project:
```

Comment:
`create_project `and `create_example` are basically the same target.

But anyway, do we really need a make target to copy-paste a template directory..?

Reply:
Let's only leave the template in the projects directory. I will modify the readme accordingly.